pkg/applicationserver/config.go

```diff
@@ -66,12 +66,17 @@ type EndDeviceFetcherCircuitBreakerConfig struct {
 	Threshold int           `name:"threshold" description:"Number of failed fetching attempts after which the circuit breaker opens"`
 }
 
+type FormattersConfig struct {
+	MaxScriptSize int `name:"max-script-size" description:"Maximum allowed size for payload formatter scripts"`
```

Comment:
While we're at it, would this also be the moment to make the script timeout configurable?